core/src/main/java/com/radixdlt/genesis/olympia/converter/resource/OlympiaNonXrdConverter.java

```diff
@@ -286,24 +293,33 @@ private static GenesisResource convertResource(
       List<OlympiaStateIR.Account> accounts,
       Decimal initialSupply,
       OlympiaStateIR.Resource resource) {
-    final var metadata =
-        ImmutableList.of(
+    final var metadataBuilder = ImmutableList.<Tuple2<String, String>>builder();
+    metadataBuilder.addAll(
+        List.of(
             tuple("symbol", resource.symbol()),
             tuple("name", resource.name()),
-            tuple("description", resource.description()),
-            tuple("url", resource.url()),
-            tuple("icon_url", resource.iconUrl()));
+            tuple("description", resource.description())));
+    if (!resource.url().isBlank()) {
+      metadataBuilder.add(tuple("info_url", resource.url()));
```

Comment:
Ditto - we need to make these URLs, not strings.

Reply:
Done